<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hsiu-Ming's Timer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="hmtimer.qrc">
    <normaloff>:/hmtimer.png</normaloff>:/hmtimer.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QSpinBox" name="spbHour"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>H&amp;our</string>
        </property>
        <property name="buddy">
         <cstring>spbHour</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spbMinute">
        <property name="maximum">
         <number>59</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelMinute">
        <property name="text">
         <string comment="minute">&amp;Min.</string>
        </property>
        <property name="buddy">
         <cstring>spbMinute</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spbSecond">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>59</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelSecond">
        <property name="text">
         <string comment="second">&amp;Sec.</string>
        </property>
        <property name="buddy">
         <cstring>spbSecond</cstring>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QCheckBox" name="chbRunRepeatedly">
        <property name="text">
         <string>Run repeatedly</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbRepeatTimes">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spbRepeatTimes">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="value">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelRepeatTimes">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string comment="(Run repeatedly N )times">times</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btnAt">
        <property name="text">
         <string comment="At a specific time">A&amp;t:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&amp;Action:</string>
        </property>
        <property name="buddy">
         <cstring>comboAction</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboAction"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>298</width>
        <height>159</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnStart">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>385</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHomepage"/>
    <addaction name="actionProject_Page"/>
    <addaction name="actionDonate"/>
    <addaction name="actionBlog"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionHomepage">
   <property name="text">
    <string>Homepage</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionProject_Page">
   <property name="text">
    <string>Project Page</string>
   </property>
  </action>
  <action name="actionDonate">
   <property name="text">
    <string>Donate</string>
   </property>
  </action>
  <action name="actionBlog">
   <property name="text">
    <string>Blog</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>spbHour</tabstop>
  <tabstop>spbMinute</tabstop>
  <tabstop>spbSecond</tabstop>
  <tabstop>chbRunRepeatedly</tabstop>
  <tabstop>chbRepeatTimes</tabstop>
  <tabstop>spbRepeatTimes</tabstop>
  <tabstop>btnAt</tabstop>
  <tabstop>comboAction</tabstop>
  <tabstop>btnStart</tabstop>
 </tabstops>
 <resources>
  <include location="hmtimer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
